functions/HttpTriggerAPIUsersIdTask/__init__.py

```diff
@@ -1,24 +1,67 @@
 import logging
-
+import pyodbc
+import os
 import azure.functions as func
+import datetime
+import json
+import time
+
 
 
+# to handle datetime with JSON
+def default(o):
+    if isinstance(o, (datetime.datetime, datetime.date)):
+        return o.isoformat()
+
 def main(req: func.HttpRequest) -> func.HttpResponse:
-    logging.info('Python HTTP trigger function processed a request.')
+    
+
+    # define the server and database names
+    server = os.environ["ENV_DATABASE_SERVER"]
+    database = os.environ["ENV_DATABASE_NAME"]
+    username = os.environ["ENV_DATABASE_USERNAME"]
+    password = os.environ["ENV_DATABASE_PASSWORD"]
+    
+    # define the connection string
+    driver = '{ODBC Driver 17 for SQL Server}'
+    cnxn = "Driver={};Server={};Database={};Uid={};Pwd={};Encrypt=yes;TrustServerCertificate=no;Connection Timeout=30;".format(
+        driver, server, database, username, password)
+    
+   
+    id = req.params.get('userId')
 
-    name = req.params.get('name')
-    if not name:
+    if not id:
+        return func.HTTPResponse('Bad Request', status_code=400 )
+    else:
         try:
-            req_body = req.get_json()
-        except ValueError:
-            pass
+            with pyodbc.connect(cnxn) as conn:
+                return tasks_query(conn, id)
+        except pyodbc.DatabaseError as err:
+            if err.args[0] == '28000':

```

Comment:
What kind of error is this? Having a numerical error present in the code with no comment makes it a bit difficult to know what's gone wrong (even if it's being returned as "Unauthorized User" below.

Reply:
I used the same error handling, maybe we add better error handling in the next Sprint. This code still needs to update a few more response codes.